### PR TITLE
Downloads: Fix autoversion expressions for Mender client packages

### PR DESCRIPTION
**Pull request by lluiscampos**
And tweak a bit the wording to both make the expressions unique for
the tool but also make it more human friendly (same as the other
tables).

### CODE REVIEW

File: 09.Downloads/docs.md [hunk 1]
```diff
@@ -258,18 +258,18 @@ airtight systems with limited access to the Internet, or when running
...
+| amd64          | Generic 64-bit x86 processors, the most popular among workstations                             | [mender-client master][mender-client_x.x.x_amd64.deb] | [mender-client master][mender-client_master_amd64.deb] (Pre-release) |
 
 <!--AUTOVERSION: "downloads.mender.io/%/"/mender "mender-client_%-1"/mender -->
 [mender-client_x.x.x_armhf.deb]: https://downloads.mender.io/master/dist-packages/debian/armhf/mender-client_master-1%2Bdebian%2Bbuster_armhf.deb
 [mender-client_x.x.x_arm64.deb]: https://downloads.mender.io/master/dist-packages/debian/arm64/mender-client_master-1%2Bdebian%2Bbuster_arm64.deb
 [mender-client_x.x.x_amd64.deb]: https://downloads.mender.io/master/dist-packages/debian/amd64/mender-client_master-1%2Bdebian%2Bbuster_amd64.deb
-<!--AUTOVERSION: "mender-client_%_a"/mender "downloads.mender.io/%/"/ignore "mender-client_%-1"/ignore -->
+<!--AUTOVERSION: "mender-client_%_a"/ignore "downloads.mender.io/%/"/ignore "mender-client_%-1"/ignore -->
```
Comment (lluiscampos) [inline 09.Downloads/docs.md:272]: This one was a mistake in #1651

File: 09.Downloads/docs.md [hunk 2]
```diff
@@ -258,18 +258,18 @@ airtight systems with limited access to the Internet, or when running
 Mender in [standalone
 mode](../02.Overview/01.Introduction/docs.md#client-modes-of-operation).
 
-<!--AUTOVERSION: "mender-client_%_a"/mender "mender-client_%-1"/mender -->
+<!--AUTOVERSION: "mender-client %][mender-client_x.x.x"/mender "mender-client %][mender-client_%_"/ignore-->
```
Comment (lluiscampos) [inline 09.Downloads/docs.md:261]: But this one I honestly don't know if it ever worked properly in `staging` and release branches :thinking:

Comment (kacf) [reply 09.Downloads/docs.md:261]: Good fix, but I think this expression is confusing enough to warrant a comment, or? I guess the intention is to keep master in the second column, while the first column is changed?

File: 09.Downloads/docs.md (hunk 1 above)
Comment (kacf) [reply 09.Downloads/docs.md:272]: Indeed, good catch!

File: 09.Downloads/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:261]: It is already existing for `mender-artifact` and `mender-cli`. This commit really just aligns the client one with the rest.

Anyway, once you point it out I'll add a comment :)

Comment (lluiscampos) [reply 09.Downloads/docs.md:263]: Done.

**Pull request merged**